common/src/main/java/com/wynntils/models/abilitytree/AbilityTreeContainerQueries.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.abilitytree;
+
+import com.google.common.collect.ImmutableMap;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.container.ScriptedContainerQuery;
+import com.wynntils.handlers.container.type.ContainerContent;
+import com.wynntils.models.abilitytree.parser.UnprocessedAbilityTreeInfo;
+import com.wynntils.models.abilitytree.type.AbilityTreeInfo;
+import com.wynntils.models.abilitytree.type.AbilityTreeNodeState;
+import com.wynntils.models.abilitytree.type.AbilityTreeSkillNode;
+import com.wynntils.models.abilitytree.type.ParsedAbilityTree;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.type.Pair;
+import com.wynntils.utils.wynn.InventoryUtils;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+
+public class AbilityTreeContainerQueries {
+    private static final int ABILITY_TREE_SLOT = 9;
+    private static final int PREVIOUS_PAGE_SLOT = 57;
+    private static final int NEXT_PAGE_SLOT = 59;
+    private static final int DUMMY_SLOT = 76; // This is the second archetype icon
+    private static final StyledText NEXT_PAGE_ITEM_NAME = StyledText.fromString("§7Next Page");
+
+    public void dumpAbilityTree(Consumer<AbilityTreeInfo> supplier) {
+        queryAbilityTree(new AbilityTreeContainerQueries.AbilityPageDumper(supplier));
+    }
+
+    public void updateParsedAbilityTree() {
```

Comment:
Where is this called from? I can't find it.

Reply:
This is only called when the tree parsing is actually used, in the next PR. This just made it into this PR (accidentally).